src/ecalc/libraries/libecalc/common/libecalc/core/result/results.py

```diff
@@ -51,6 +51,7 @@ class PumpResult(GenericComponentResult):
     inlet_liquid_rate_m3_per_d: TimeSeriesRate
     inlet_pressure_bar: TimeSeriesFloat
     outlet_pressure_bar: TimeSeriesFloat
+    operational_head: Optional[List[float]]
```

Comment:
bruk timeseries som over.

Reply:
helst ikke optional, hvis mulig. er den optional?

Reply:
No need to be optional, will change and use timeseries as well.

Reply:
ser du har laga timeseries over rett, men her har du angitt at det er liste

Reply:
Ja, den var liste - men oppdateres nå